pkg/config/config.go

```diff
@@ -148,6 +151,30 @@ func getConfig(configKey, defaultValue string) (result string) {
 	return
 }
 
+// SetKernelSourceDir sets the directory for all kernel source. This is used for bcc. Only the top level directory is needed.
+func SetKernelSourceDir(dir string) {
```

Comment:
should this function return an error if the `root` dir is invalid? 

```suggestion
func SetKernelSourceDir(dir string) error {

fileInfo, err := os.Stat(path)
if err != nil {
  return err
}

if !fileInfo.IsDir() {
   return fmt.Errorf("expected  kernel root path %s to be a directory", dir)
}
```

Also how we rename the function to `func SetKernelRootDir`